pkg/helm/actions/uninstall_release_test.go

```diff
@@ -0,0 +1,94 @@
+package actions
+
+import (
+	"errors"
+	"io/ioutil"
+	"testing"
+
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chartutil"
+	kubefake "helm.sh/helm/v3/pkg/kube/fake"
+	"helm.sh/helm/v3/pkg/release"
+	"helm.sh/helm/v3/pkg/storage"
+	"helm.sh/helm/v3/pkg/storage/driver"
+)
+
+func TestUninstallRelease(t *testing.T) {
+	tests := []struct {
+		name    string
+		release *release.Release
+		err     error
+	}{
+		{
+			name: "successful release uninstall should remove release and dependant resources",
```

Comment:
do we really check in this test that dependant resources are removed? If not, we should adjust the description.

Reply:
we don't check dependant resources in this test, updated the description